wp-content/themes/chrisdinh-photography/resources/views/components/slider.blade.php

```diff
@@ -1,27 +1,53 @@
-            <ul class="splide__list">
-                @foreach ($data as $item)
-                    <li class="splide__slide relative">
-                        <picture class="block relative w-full h-auto">
-                            <source srcset="{{ wp_get_attachment_image_srcset( $item['homepage__slide-image-desktop']['ID'] ) }}" media="(min-width: 50em)">
-                            <source srcset="{{ wp_get_attachment_image_srcset( $item['homepage__slide-image-mobile']['ID'] ) }}">
-                            <img alt="{{ $item['homepage__slide-image-mobile']['alt'] }}" class="relative w-full h-full object-cover">
-                        </picture>
-                    </li>
-                @endforeach
-            </ul>
+        <div class="flex justify-center items-center">
+            <div class="splide__arrows z-10">
+                <button class="splide__arrow splide__arrow--prev !bg-transparent z-10">
+                    <!--using the same svg path because the button is configured to set the svg path backwords for the left button-->
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" stroke="white" class="bi bi-chevron-right" viewBox="0 0 16 16"> <path fill-rule="evenodd" d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"/> </svg>
+                <button class="splide__arrow splide__arrow--next !bg-transparent z-10">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" stroke="white" class="bi bi-chevron-right" viewBox="0 0 16 16"> <path fill-rule="evenodd" d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z"/> </svg>
+                </button>
+            </div>
+            <div class="splide__track justify-center items-center">
+                <ul class="splide__list ">
+                    @foreach ($data as $item)
+                    <!--figure out how to get the 4th image to fill the container -->
+                        <li class="splide__slide relative flex items-center justify-center">
+                            <picture class="block relative w-full h-auto">
+                                <source srcset="{{ wp_get_attachment_image_srcset( $item['homepage__slide-image-desktop']['ID'] ) }}" media="(min-width: 50em)">
+                                <source srcset="{{ wp_get_attachment_image_srcset( $item['homepage__slide-image-mobile']['ID'] ) }}">
+                                <img alt="{{ $item['homepage__slide-image-mobile']['alt'] }}" class="w-full h-full object-cover object-center">
+                            </picture>
+                        </li>
+                    @endforeach
+                </ul>
+            </div>
+            <!--handles color of pagination text-->
+            @if (get_field('slider__custom-pagination'))
```

Comment:
Use `$sliderSettings` instead

Reply:
updated in commit 7a8dd48d77f9683e02904cbb06e5dc28970869e9

Reply:
updated in commit 1569f54c28e33513007525a18dde9d946fba34c1